src/PHPSemVerCheckerGit/ProcessedFileList.php

```diff
@@ -40,20 +54,35 @@ public function getScanner()
         return $this->scanner;
     }
 
+    /**
+     * @return string[]
+     */
+    public function getFiltered()
+    {
+        return $this->filtered;
+    }
+
     /**
      * @return int
      */
-    public function getOriginalAmount()
+    public function getFilteredAmount()
```

Comment:
Use "count" instead of "amount" (see https://english.stackexchange.com/questions/141564/differences-between-amount-count-number-and-quantity)

Reply:
very useful, thank you.